index.js

```diff
@@ -10,15 +13,32 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     this.addonOptions = app.options[this.name] || {};
-    this.import('vendor/lazysizes.js');
+    this.plugins = this.addonOptions.plugins || [];
+    delete this.addonOptions.plugins;
+
+    this.plugins.forEach((plugin) => {
+      this.import(`vendor/plugins/${plugin}/ls.${plugin}.js`);
+    });
+
+    this.import(`vendor/lazysizes.js`);
   },
 
-  treeForVendor() {
-    let lazySizesPath = path.dirname(require.resolve('lazysizes'));
+  treeForVendor(tree) {
+    let trees = [tree];
+
+    let lazyNodeTree = this.treeGenerator(path.dirname(require.resolve('lazysizes')));
+    if (this.plugins.length > 0) {
+      let include = [];
+      this.plugins.forEach((plugin) => {
+        include.push(`plugins/${plugin}/ls.${plugin}.js`);
+      });
```

Comment:
Minor stylistic issue: this could be a bit cleaner using `.map()`:

```
let include = this.plugins.map((plugin) => `plugins/${plugin}/ls.${plugin}.js`);
```